app/main.py

```diff
@@ -1,3 +1,19 @@
+import math
+
+
 class OnlineCourse:
-    # write your code here
-    pass
+    def __init__(self, name, description, weeks):
+        self.name = name
+        self.description = description
+        self.weeks = weeks
+
+    @staticmethod
+    def days_to_weeks(days):
+        return math.ceil(days / 7)
+
+    @classmethod
+    def from_dict(cls, course_dict):
+        return OnlineCourse(
```

Comment:
It'd be better to use cls to create a new instance.
`return cls(name=course_dict["name"], ... )`